seminar2.py

```diff
@@ -0,0 +1,35 @@
+import fractions
+
+
+# Напишите программу, которая получает целое число и возвращает его шестнадцатеричное строковое представление.
+# Функцию hex используйте для проверки своего результата.
+
+def task1():
+
+    i = int(input("Введите число: "))
+    h = (format(i, '#x'))

```

Comment:
Хитро, Однако ваш код хуже масштабируется.
```suggestion
    h = format(i, '#x')
```